iibb.py

```diff
@@ -22,7 +22,8 @@
 __license__ = "LGPL-3.0-or-later"
 __version__ = "3.01b"
 
-import md5, os, sys, tempfile, traceback
+import os, sys, tempfile, traceback
+from hashlib import md5
```

Comment:
Aca se podría soprtar python 2 y 3 capturando la excepción:
```suggestion
try:
    from hashlib import md5
except ImportError:
    from md5 import md5
```